lib/gambit/unstable/unstable.scm

```diff
@@ -0,0 +1,60 @@
+;;;============================================================================
+
+;;; File: "unstable.scm"
+
+;;; Copyright (c) 1994-2021 by Marc Feeley, All Rights Reserved.
+
+;;;============================================================================
+
+;;; Experimental procedures. May be changed or removed at any time.
+
+(##include "unstable#.scm")
+
+;;;----------------------------------------------------------------------------
+
+;; TODO: The following are copied from lib/_num#.scm
+
+(##define-macro (macro-inexact--1) -1.0)
+
+(##define-macro (macro-make-exact-10^n-table)
+
+  (define max-exact-power-of-10 22)
+  ;; (floor (inexact->exact (/ (log (expt 2 (macro-flonum-m-bits-plus-1)))
+  ;;                           (log 5))))
+
+  (let ((t (make-vector (+ max-exact-power-of-10 1))))
+
+    (let loop ((i max-exact-power-of-10))
+      (if (not (< i 0))
+          (begin
+            (vector-set! t i (exact->inexact (expt 10 i)))
+            (loop (- i 1)))))
+
+    `',(list->f64vector (vector->list t))))
+
+(define exact-10^n-table (macro-make-exact-10^n-table))
+
+(define-prim&proc (make-inexact-real (negative? boolean)
```

Comment:
I'm not sure why `make-inexact-real` is not accessible.  Compiling with `gsc -expansion ...` will reveal what name is actually being used.

Reply:
`gsc -expansion unstable.scm` shows that `(define-prim&proc (make-inexact-real ...) ...)` is not expanded, even if changed to `##define-prim&proc`.

Reply:
Of course! There's a `(##include "~~lib/_gambit#.scm")` needed at the top of the file because `define-prim&proc` is a macro defined there.

Reply:
Right. `(include "unstable.scm")` was also needed instead of `(include "unstable#.scm")`.